### ISSUE TITLE
Incorrect redstone warning

### ISSUE THREAD

**Issue by duncanwebb** [TeamPneumatic/pnc-repressurized#354]
### Minecraft Version
1.12.2

### Forge Version
1.12.2-14.23.5.2768

### Mod Version
1.12.2-0.11.3-366

### Describe your problem, including steps to reproduce it
Took me a while to figure this out, as you can see from the image there is a warning about redstone and there is a redstone signal being applied. However the pressure is increasing so it is working.

First I thought that it was a problem with automated redstone emitters but I don't think it is. 

To reproduce the situation first set the pressure settings in pressure gauge tube module with an advanced circuit, let the pressure reach the settings and then change from greater than to less than. I also changed the low/high redstone settings.

![strange-message](https://user-images.githubusercontent.com/4198031/60203911-00373580-984e-11e9-9fd5-9bfaf5af9a11.png)

Hope this is clear and many thanks for all you efforts. Like the cheaper advanced circuits and the jetboots display even though the letters overlap a bit.

**Comment by desht** [comment]
I'll look into the redstone problem; could well be a sync issue.

Regarding text overlapping, do you have a screenshot of that? Sounds like something that should be easy to adjust...

**Comment by duncanwebb** [comment]
![jet-boots-headsup](https://user-images.githubusercontent.com/4198031/60207232-ac304f00-9855-11e9-8aaf-d642a8f067b5.png)

I'm using smooth font as I find it easier to read. Smooth font seems to be a new trend.

**Comment by desht** [comment]
Yep, smooth font is probably throwing off the text width calculations. I'll see if there's a clean fix for it, or failing that a dirty hack..

**Comment by desht** [comment]
Jetboots stat window should now render better for you.

I could not reproduce the redstone warning issue - tried numerous combinations of signal/no signal/high/low and it always behaved as I expected.  Will leave this open for now though.

**Comment by MuteTiefling** [comment]
![Capture](https://user-images.githubusercontent.com/9543430/71133026-5666af00-21c7-11ea-8723-ddb0d1ab1eb4.PNG)

I've been noticing this inversion as well. The machine functions as expected, but the warning is there when it's working, rather than when it's disabled. Hard to tell from the screenshot, but it is using the power in the buffer and definitely making air.

This happens on everything as far as I can tell. 

Forge: 14.23.2847
PNC 1.12.2-0.11.7-375

I suspect something has just gone wrong in the world though.

I can't reproduce with only PNC. 
I've also added a few of the major mods in my pack to my test profile (thermal series, crafttweaker) and am not able to reproduce it there either. 
Worse, I can't reproduce it in the my actual pack in a new world. Maybe it's just a byproduct of using that world for so much testing while swapping mods in and out.

**Comment by desht** [comment]
Yeah, there's something really subtle going on here...
